rpcs3/Emu/Cell/lv2/sys_memory.h

```diff
@@ -50,10 +50,7 @@ struct lv2_memory_container
 	static const u32 id_step = 0x1;
 	static const u32 id_count = 16;
 
-	// This is purposely set lower to fake the size of the OS
-	// Todo: This could change with requested sdk
-	const u32 size = 0xEC00000; // Amount of "physical" memory in this container
-
+	u32 size; // Amount of "physical" memory in this container
```

Comment:
It should remain const since it cannot change. Just need to remove default constructor.